src/libs/actions/Report.js

```diff
@@ -616,6 +617,29 @@ function createOptimisticChatReport(participantList) {
     };
 }
 
+/*
+ * Creates an optimistic IOU report with a randomly generated reportID
+ */
+function buildOptimisticIOUReport(total, chatReportID, currency, locale) {
+    const formattedTotal = NumberFormatUtils.format(locale,
+        total, {
+            style: 'currency',
+            currency,
+        });
+    return {
+        cachedTotal: formattedTotal,
+        chatReportID,
+        currency,
+        hasOutstandingIOU: true,
+        managerEmail: '__FAKE__',
+        ownerEmail: '__FAKE__',
+        reportID: ReportUtils.generateReportID(),
+        state: 'SUBMITTED',
```

Comment:
Can we create a CONST for this if it doesn't exist yet?

Reply:
We don't have any consts for the states yet, so I was hesitant to create a new REPORT_STATE object. Also, we talked about removing report state from the App. 

But again, I'm happy to add if you still think we should.

Reply:
Where did we talk about removing state from App? I think we should add it. Worst case, we remove it later when we remove state from App.

Reply:
I am +1 for adding this as const to the App and we can remove that easily at any point if we decided to do so

Reply:
It was [discussed here](https://expensify.slack.com/archives/C03TQ48KC/p1662462104472869) and had pretty wide approval. But I'll do this now, no problem